Assets/Mirror/Runtime/Transport/TelepathyTransport.cs

```diff
@@ -96,7 +103,14 @@ public void LateUpdate()
         // server
         public override bool ServerActive() => server.Active;
         public override void ServerStart() => server.Start(port);
-        public override bool ServerSend(int connectionId, int channelId, byte[] data) => server.Send(connectionId, data);
+        public override bool ServerSend(int connectionId, int channelId, ArraySegment<byte> segment)
+        {
+            // telepathy doesn't support allocation-free sends yet.
+            // previously we allocated in Mirror. now we do it here.
+            byte[] data = new byte[segment.Count];
```

Comment:
This makes things is a lot worse for Telepahy,  Suppose you have 50 observers,  now you allocate 50 byte[] for the same message.  Before we only allocated 1

This is in fact the reason why I have no sent his PR already